out/notebooks/spl2/installer.ts

```diff
@@ -478,11 +487,11 @@ async function promptToDownloadLsp(alsoInstallJava: boolean): Promise<boolean> {
  */
 export async function getLatestSpl2Release(context: ExtensionContext, progressBar: StatusBarItem): Promise<void> {
     return new Promise(async (resolve, reject) => {
-        const lspArtifactPath = path.join(context.globalStorageUri.fsPath, 'lsp');
+        const lspArtifactPath = getLocalLspDir(context);
         // TODO: Remove this hardcoded version/update time and check for updates
-        let lspVersion: string = '2.0.362'; // context.globalState.get(stateKeyLatestLspVersion) || "";
+        let latestLspVersion: string = '2.0.362'; // context.globalState.get(stateKeyLatestLspVersion) || "";
         const lastUpdateMs: number = Date.now(); // context.globalState.get(stateKeyLastLspCheck) || 0;
```

Comment:
Should this be Date.now?

Reply:
This is my way of turning the feature off for now until I have a chance to flesh out the issues I was seeing with retrieving the latest version from maven metdata, related to the `TODO` above